src/components/widgets-settings/dates-picker/dates-picker.directive.coffee

```diff
@@ -4,12 +4,13 @@ module.directive('settingDatesPicker', ($templateCache, $filter, ImpacWidgetsSvc
   return {
     restrict: 'A',
     scope: {
-      parentWidget: '='
-      deferred: '='
+      parentWidget: '=?'
+      deferred: '=?'
```

Comment:
deferred should be compulsory in my opinion, to be consistent with the others settings logic, until its refactored.